chromadb/utils/embedding_functions.py

```diff
@@ -676,8 +677,17 @@ def __call__(self, input: Documents) -> Embeddings:
                 self._api_url, json={"instances": [{"content": text}]}
             ).json()
 
-            if "predictions" in response:
-                embeddings.append(response["predictions"]["embeddings"]["values"])
+            predictions = response.get('predictions')
+            if predictions:
+                for prediction in predictions:
+                    embeddings.append(prediction["embeddings"]["values"])
+            else:
+                if predictions is None:
+                    raise KeyError(f'Key `predictions` of the response from Vertex Embedding API is not existed!! Please confirm the response. \n{response}')
```

Comment:
```suggestion
                    raise KeyError(f'Something went wrong -- `predictions` from the Vertex Embedding API does not exist. Response: {response}')
```

Reply:
I will add this wording to replace the original one.